ui/app/utils/default-sentinel-policy-templates.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+// TODO: Several of these are placeholders. Fill them out with real policies or remove them.
+
+import countLimitsPolicy from './sentinel_policy_templates/count-limits';
+import noFridaysPolicy from './sentinel_policy_templates/no-friday-deploys';
+import alwaysFailPolicy from './sentinel_policy_templates/always-fail';
+// import enforceServiceChecksPolicy from './sentinel_policy_templates/enforce-service-checks';
+import canariesOnlyPolicy from './sentinel_policy_templates/canaries-only';
+// import enforceMeshPolicy from './sentinel_policy_templates/enforce-service-mesh';
+// import metadataPolicy from './sentinel_policy_templates/metadata-required';
+// import consulSDPolicy from './sentinel_policy_templates/consul-service-discovery-only';
+// import vaultSecretsPolicy from './sentinel_policy_templates/vault-secrets-only';
+// import dyanmicPortsPolicy from './sentinel_policy_templates/dynamic-ports-only';
+import resourceLimitsPolicy from './sentinel_policy_templates/resource-limits';
+// import constraintEnforcmentPolicy from './sentinel_policy_templates/constraint-enforcement';
```

Comment:
A big TODO with this PR: do we launch this without the commented-out policies (which are currently just placeholders below), or can we write them in such a way that they make sense as starter templates (knowing Nomad ships with Sentinel v0.15.x?)